frame/referenda/src/types.rs

```diff
@@ -161,9 +128,9 @@ pub struct TrackInfo<Balance, Moment> {
 	/// Minimum aye votes as percentage of overall conviction-weighted votes needed for
 	/// approval as a function of time into decision period.
 	pub min_approval: Curve,
-	/// Minimum turnout as percentage of overall population that is needed for
```

Comment:
there seems to be a `TODO` in `name` field.

Reply:
don't understand. where's the TODO?

Reply:
https://github.com/paritytech/substrate/blob/18c6438daff149e7f847797f51f14cbe74839fc3/frame/referenda/src/types.rs#L113

Reply:
Not introduced here but fixed anyway.